test/test_sparse.py

```diff
@@ -1503,6 +1503,14 @@ def test_log1p(self):
             device=self.device)
         self._test_log1p_tensor(input, torch.zeros([5, 6, 0]))
 
+    def test_mv(self):
+        i = self.index_tensor([[0, 1, 1], [2, 0, 2]])
+        v = self.value_tensor([3, 4, 5])
+        x = self.sparse_tensor(i, v, torch.Size([2, 3]))
+        y = torch.ones(3, device=self.device)
+
+        self.assertEqual(self.value_tensor([3, 9]), x.matmul(y))
+
```

Comment:
We should probably have test cases that make sure your asserts are triggered correctly, with `self.assertRaisesRegex`.

Reply:
Sorry, I don't use `self.assertRaisesRegex` before. So I don't how and why to use this.

Reply:
`self.assertRaisesRegex` is for checking to make sure functions raise exceptions when expected. The way to use it is:
```
with self.assertRaisesRegex(<type of exception>, <regex string that matches the expected error message>):
    <call the function with arguments that will cause the expected exception>
```

The error messages you're regex-matching against are the messages you wrote in your `TORCH_CHECK` calls. There are many examples in `test_sparse.py`.

Reply:
Done. Thank you for your detail explain.